packages/manager/src/views/WebPropertyDetailPage/WebPropertyDetailPage.tsx

```diff
@@ -58,27 +62,32 @@ const URL_LENGTH_LIMIT = 50;
 export const WebPropertyDetailPage = (): JSX.Element => {
   const { query } = useRouter();
   const [isRowExpanded, setIsRowExpanded] = useState<Record<string, boolean>>({});
+  const [searchTerm, setSearchTerm] = useState('');
+  const [filtervalue, setFilter] = useState('');
```

Comment:
change this to a better name like filterByEnv and setFilterByEnv as this name is too generic